File: C/2061.c
#include <stdio.h>

int main(){
    int abas, acoes, i;
    char acao[7];
    scanf("%d %d", &abas, &acoes);
    
    for(i = 0; i < acoes; i++) {
        scanf("%s", acao);
        if(acao[0] == 'f') abas++;
        else abas--;
    }
    
    printf("%d\n", abas);
    return 0;
}

File: javascript/2061.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var [abas, acoes] = lines.shift().split(' ').map(e => parseInt(e))

for(let i = 0; i < acoes; i++) {
    var acao = lines.shift()
    if(acao[0] == 'f') abas++
    else abas--
}

console.log(abas)